### PR TITLE
Switch to picocli

### PR DESCRIPTION
**Pull request by Nv7-GitHub**
This uses CLI arguments instead of `stdin`.  For example, you can do 
```sh
<executable> assets/dna/random/dnalong.fa --amino=ser
```
or
```sh
<executable> assets/dna/random/dnalong.fa --amino=ser --min=0 --max=100
```

Help message:
```
Usage: DNAnalyzer [-hV] --amino=<aminoAcid> [--max=<maxCount>]
                  [--min=<minCount>] DNA
A program to analyze DNA sequences.
      DNA                   The FASTA file to be analyzed.
      --amino=<aminoAcid>   The amino acid representing the start of a gene.
  -h, --help                Show this help message and exit.
      --max=<maxCount>      The maximum count of the reading frame.
      --min=<minCount>      The minimum count of the reading frame.
  -V, --version             Print version information and exit.
```

### CODE REVIEW

**Comment by Nv7-GitHub** [comment]
@Verisimilitude11 can you make sure `CmdArgs.java` is correct?

**Review by Verisimilitude11 — APPROVED** [review]
Looks good! I'll merge this after merging #125. I might need your help in fixing any conflicts that may occur.

**Comment by Verisimilitude11** [comment]
To be clear, you want me to keep the selected text, right: 
image

@Nv7-GitHub

**Pull request merged**
